crypto/crypto.go

```diff
@@ -72,7 +72,7 @@ func (client *Crypto) Config() interface{} {
 }
 
 func (client *Crypto) setupFSBackend(config core.ServerConfig) error {
-	log.Logger().Infof("Setting up FileSystem backend for storage of private key material. " +
+	log.Logger().Info("Setting up FileSystem backend for storage of private key material. " +
 		"Discouraged for production use unless backups and encryption is properly set up. If not, consider using the Vault backend.")
```

Comment:
Remove the "If not" and add Hashicorp. Just "consider using the Hashicorp Vaul backend"